docs/examples/multimodal_classification/Multimodal_Classification_with_Ludwig_Python_API.ipynb

```diff
@@ -0,0 +1,388 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dqW_BZ60kyWP"
+      },
+      "source": [
+        "# Multimodal Classification of Twitter Bots - Python API\n",
+        "\n",
+        "*We recommend using a GPU runtime for this example. In the Colab menu bar, choose **Runtime** > **Change Runtime Type** and choose **GPU** under Hardware Accelerator.*\n",
+        "\n",
+        "In this notebook, we will show how to use the Ludwig Python API to:\n",
+        "\n",
+        "\n",
+        "1.   [Train a Ludwig Model](#scrollTo=Train)\n",
+        "2.   [Evaluate the trained model](#scrollTo=Evaluate)\n",
+        "2.   [Visualize training and test metrics](#scrollTo=Visualize_Metrics)\n",
+        "\n",
+        "This example is uses a dataset from Kaggle, so you'll need a [Kaggle account](https://www.kaggle.com/account/login) to download it."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "BUVNRi_rCAFp"
+      },
+      "source": [
+        "# Upload Kaggle Credentials\n",
+        "\n",
+        "If you have a kaggle.json file, paste its contents into the code cell below, then run to write your kaggle API key into the Colab environment."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "fw8YpR1TCAee"
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "\n",
+        "# Replace this string with the contents of your kaggle.json file.\n",
+        "KAGGLE_JSON = \"\"\"\n",
+        "{\"username\":\"\",\"key\":\"\"}\n",
```

Comment:
Is there another way we can have the user provide their Kaggle credential instead of typing it directly into the notebook?  If the user typed their kaggle credentials and then saved the notebook in their own GH repo, their Kaggle credentials will be potentially visible to others.    

I did just that saving it to this location in my personal GH repo: https://github.com/jimthompson5802/ludwig_examples/blob/master/Multimodal_Classification_with_Ludwig_CLI.ipynb

<img width="557" alt="image" src="https://user-images.githubusercontent.com/1425269/164121390-9f94b4cf-41d3-4425-a860-d561a18b3802.png">

From what I can see, whenever the colab notebook runs, the current working directory is `/content`.  Maybe we can tell the user to upload their credentials to this directory and have the code read it from there.

I believe this is should in the docker container and should be private to the user.  When the run-time is terminated, the contents is deleled.  Not sure how we can verify this.

Reply:
I share these concerns, but this is the best idea I've got.

Another option is described here, where you upload Kaggle.json to google drive, and mount gDrive in colab.  This is arguably a better process to use as a developer, but I felt like this is too many extra steps just to get a tutorial running.  https://www.analyticsvidhya.com/blog/2021/06/how-to-load-kaggle-datasets-directly-into-google-colab/

Reply:
Oh hey, how about this one:

https://www.kaggle.com/general/74235

```python
from google.colab import files

files.upload()
```

shows an upload prompt, we could have the user upload their kaggle.json then move it to the correct location.

Reply:
I went ahead and implemented the `files.upload()` version.  What do you think?

Reply:
I just tested the `files.upload()` and it worked and it does not expose the user's kaggle credentials.  This should work.

One question, though, what browser are you using?  I'm using Safari and I had to [RTFM](https://www.lifewire.com/display-hidden-files-in-os-x-open-and-save-dialog-boxes-2260878) to figure out how to display the hidden `.kaggle` directory to be able to select `kaggle.json` file for upload.  The magic sequence with Safari is the CMD+SHIFT+.(period)  key sequence.

Other than this, looks like it is working.    LGTM.